packages/common/src/dto/api/index.ts

```diff
@@ -154,10 +154,14 @@ export interface IUserProfile {
   email: string;
   username: string;
 }
-export interface IDevWorkspacePreferences {
+export interface IWorkspacePreferences {
   'skip-authorisation': GitProvider[];
-  [key: string]: string | string[];
+  'trusted-sources'?: TrustedSources;
+  [key: string]: unknown;
 }
+export type TrustedSourceAll = 'all';
```

Comment:
IMO semantically it would be better to use `*`

Reply:
I'll change that in the next commit.

Reply:
done